lib/museum.rb

```diff
@@ -0,0 +1,38 @@
+class Museum
+  attr_reader :name,
+              :exhibits,
+              :patrons
+  def initialize(name)
+    @name = name
+    @exhibits = []
+    @patrons = []
+  end
+
+  def add_exhibit(new_exhibit)
+    @exhibits << new_exhibit
+  end
+
+  def recommend_exhibits(patron)
+    @exhibits.find_all do |exhibit|
+      patron.interests.any?(exhibit)
```

Comment:
This doesn't work as laid out in the spec because @exhibits should be an object and `patron.interests` is an array of strings. so in order to compare it would need to be:
```suggestion
      patron.interests.any?(exhibit.name)
```